toolchain/check/testdata/function/generic/cross_library_name_collision_public.carbon

```diff
@@ -0,0 +1,567 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
```

Comment:
For the check test here, can you please add `ranges=only`?

Reply:
Addressed by deleting the test :)